gcc/rust/hir/rust-hir-dump.h

```diff
@@ -32,6 +32,7 @@ class Dump : public HIRFullVisitor
 public:
   Dump (std::ostream &stream);
   void go (HIR::Crate &crate);
+  virtual void visit (AST::Attribute &attribute);
```

Comment:
```suggestion
  void visit (AST::Attribute &attribute);
```

Also should be private.